gcloud/bigquery/table.py

```diff
@@ -693,6 +699,225 @@ def insert_data(self,
 
         return errors
 
+    def upload_from_file(self,  # pylint: disable=R0913,R0914
```

Comment:
RE: 80 cols on using the full names instead of the pylint error codes, you can use `disable=` before the `def` and `enable=` after (I mentioned elsewhere)